src/main/java/org/team1540/bunnybotTank2023/io/indexer/IndexerIOReal.java

```diff
@@ -0,0 +1,35 @@
+package org.team1540.bunnybotTank2023.io.indexer;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import org.team1540.bunnybotTank2023.Constants;
+
+public class IndexerIOReal implements IndexerIO {
+    private final CANSparkMax topMotor = new CANSparkMax(Constants.IndexerConstants.TOP_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private final CANSparkMax bottomMotor = new CANSparkMax(Constants.IndexerConstants.BOTTOM_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
+    public IndexerIOReal() {
+        topMotor.setIdleMode(CANSparkMax.IdleMode.kBrake);
+        bottomMotor.setIdleMode(CANSparkMax.IdleMode.kBrake);
+    }
```

Comment:
You'd probably want to specify inversion here. Also, because these motors are NEO 550s, you will _really_ want a current limit here (I'd say about 30 amps), unless you like the smell of magic smoke.